aatools/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""aa-tools: Toolbox for Anticipatory Action.
+
+The ``aatools`` library contains modules designed to assist with
+downloading and processing the data required for climate- and weather-related
```

Comment:
nitpicking, but wouldn't limit it to climate- and weather-related aa for the future